apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -134,6 +134,11 @@ def get_query_server_config(name='beeswax', connector=None):
           cache.set("llap", json.dumps({"host": LLAP_SERVER_HOST.get(), "port": LLAP_SERVER_THRIFT_PORT.get()}), CACHE_TIMEOUT.get())
       activeEndpoint = json.loads(cache.get("llap"))
     elif name != 'hms' and name != 'impala':
+      Endpoint = cache.get("hiveserver2")
+      if Endpoint is None:
+        activeEndpoint = None
+      else:
+        activeEndpoint = Endpoint.get(username)
```

Comment:
`username`

Where is this variable defined? Can we use user.id instead as in [https://github.com/cloudera/hue/pull/1005/files#diff-05088e6a157c3c3047871f25b14f0ff1R80](url). Usernames can be deleted & recreated.

Reply:
username = user.username